Neos.ContentRepository.Core/Classes/Feature/Common/ConstraintChecks.php

```diff
@@ -261,6 +257,26 @@ protected function requireConstraintsImposedByAncestorsAreMet(
                 }
             }
         }
+
+        if (!in_array(false, $autoCreatedParentNodeAggregates, true)) {
```

Comment:
im not sure what should happen, if only some parents are autocreated 😂 crazy shit